src/denostore.ts

```diff
@@ -158,24 +162,26 @@ export default class Denostore {
       });
     }
 
-    //handles posted query and responds
+    // where GraphQL queries are handled
     this.#router.post(this.#route, async (ctx: Context): Promise<void> => {
       const { response, request } = ctx;
       try {
         const { query, variables } = await request.body().value;
-        console.log(query, variables);
 
-        //caching happens inside of resolvers (nested within schema, so graphql func invocation)
-        const results = await graphql({
+        // resolve GraphQL query
+        const graphqlResults = await graphql({
           schema: this.#schema,
           source: query,
+          // pass denostore instance through context to use methods in resolvers
           contextValue: { denostore: this },
           variableValues: variables,
         });
+
         // if errors delete results data
-        results.errors ? delete results.data : null;
-        response.status = results.errors ? 500 : 200;
-        response.body = results;
+        graphqlResults.errors ? delete graphqlResults.data : null;
```

Comment:
should this just be an "if" statement instead of ternary?

Reply:
@Jamesmjkim was there a specific reason you used a ternary here? Was it to match the line below it?

Reply:
@sktatsuno No particular reason. I just thought ternary looked clean here. You can just use "if" statement if you want.